gammapy/estimators/map/excess.py

```diff
@@ -68,7 +68,7 @@ class ExcessMapEstimator(Estimator):
     If a model is set on the dataset the excess map estimator will compute the
     excess taking into account the predicted counts of the model.
 
-    ..note::
```

Comment:
I think using a `note` is correct, there is just a space missing? `.. note::` ?

https://docutils.sourceforge.io/docs/ref/rst/directives.html#note

Reply:
Yes, my bad...